Main.java

```diff
@@ -9,7 +10,21 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
+    // create a Scanner for user input 
+    Scanner input = new Scanner (System.in);
+    System.out.println ("Please enter in two integers,on seperate lines, to divide.");
+    //declare a variable to store first
+
+    int first = input.nextInt ();
+    //declare a variable to store second
+    int second = input.nextInt ();
+    //declare a variable to store result 
+  
```

Comment:
This is good looking code. Just make sure that the line comments are actually attached to the line they are commented (i.e. no blank space between the comment and the line it belongs with). This doesn't affect its ability to run, but it is more of a style thing!